src/app.rs

```diff
@@ -355,6 +372,33 @@ impl Application for AppModel {
                     .height(Length::Fill)
                     .into()
             }
+            Some(Page::Motherboard) => {
+                let Some(dmidecode) = &self.dmidecode else {
+                    return widget::text::title1(fl!("error-occurred")).into();
+                };
+
+                if let Some(dmidecode_str) = &self.dmidecode {
+                    if dmidecode_str.starts_with(fl!("error-occurred").as_str()) {
+                        return widget::text::title1(dmidecode_str).into();
+                    } else {
+                        let dmidecode = dmidecode
+                            .lines()
+                            .map(|line: &str| {
+                                let (prefix, suffix) = line.split_once(':').unwrap();
```

Comment:
This wont work for this string, we need to exclude lines without `:` first.

Reply:
Mm I'm not sure how to do that personally lol. I just copied the other lines.

Reply:
Yeah me neither, but maybe there's a way to prompt the user for authorization and elevate the process.

Reply:
I'm sure there is/will be but perhaps we'll want to do this a different way.

Reply:
Sorry I confused the conversations, you can use `filter` for that.